build.sbt

```diff
@@ -2,6 +2,12 @@ name := "root"
 
 Common.settings
 
+libraryDependencies ++= Seq(
```

Comment:
I think projects themselves if they are going to run on their own servers unless we want to put the entire project on each box then i think top lvl is the way to go.